torchvision/datasets/mnist.py

```diff
@@ -206,15 +206,15 @@ class FashionMNIST(MNIST):
         target_transform (callable, optional): A function/transform that takes in the
             target and transforms it.
     """
+    mirrors = [
+        "http://fashion-mnist.s3-website.eu-central-1.amazonaws.com/"
+    ]
```

Comment:
`(Fashion|K)MNIST` use the same `download` logic as `MNIST`. Thus, we need to also provide the URLs in the same fashion (no pun intended :wink:) although we only have a single mirror.